@here/harp-mapview/lib/MapView.ts

```diff
@@ -2234,6 +2249,7 @@ export class MapView extends THREE.EventDispatcher {
         );
         this.m_camera.near = clipPlanes.near;
         this.m_camera.far = clipPlanes.far;
+        this.m_maxVisibility = clipPlanes.maximum;
```

Comment:
what is the difference between maximum and far? Can maximum be further than far, or is far up to maximum?

Reply:
Visibility range (maxVisibility) is the absolutely maximum distance that far plane may be put on. It is our internal limit for distant rendering. We need it for situations when camera may be tilt with look at vector close to parallel to the ground, in this conditions simple math would account for infinite distance.